### PR TITLE
Add computer man

### PR DESCRIPTION
**Pull request by ValokR**

### CODE REVIEW

**Comment by ValokR** [comment]
Please review, @casey-collab

File: english/Computer-Man/Computer-Man.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+You're walking down the street, and the glint of a flashy new computer
```
Comment (casey-collab) [inline english/Computer-Man/Computer-Man.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: english/Computer-Man/Computer-Man.md (hunk 1 above)
Comment (ValokR) [inline english/Computer-Man/Computer-Man.md]: @casey-collab

**Comment by ValokR** [comment]
@casey-collab

File: english/Computer-Man/Computer-Man.md (hunk 1 above)
Comment (casey-collab) [inline english/Computer-Man/Computer-Man.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by ValokR** [comment]
@casey-collab

File: english/marshmallow.md [hunk 2]
```diff
@@ -1138,4 +1138,7 @@ Let's learn some [mandarin](learn-mandarin/learn-mandarin.md)
 Neurons for social memories, [more info](http://news.mit.edu/2016/scientists-identify-neurons-social-memory-0929)
 
 What is the meaning of life?
-To find out click [here](meaning-of-life/meaning-of-life.md).
\ No newline at end of file
+To find out click [here](meaning-of-life/meaning-of-life.md).
+
+What is that shiny in the window?  Come in and take a
+[look](Computer-Man.md)
```
Comment (casey-collab) [inline english/marshmallow.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/Computer-Man/Computer-Man.md (hunk 1 above)
Comment (casey-collab) [inline english/Computer-Man/Computer-Man.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by ValokR** [comment]
@casey-collab

**Comment by casey-collab** [comment]
Looks great! :shell:

**Pull request merged**